Makefile

```diff
@@ -51,85 +60,182 @@ ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=shentu \
 		  -X github.com/cosmos/cosmos-sdk/version.AppName=shentud \
 		  -X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
 		  -X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
-		  -X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep)"
+		  -X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep)" \
```

Comment:
At line 55, should be `whitespace := $(whitespace) $(whitespace)`
Or it will cause error on `$build_tags_comma_sep` and `make build-static-linux-amd64`

Reply:
fix